common/network-yarn/pom.xml

```diff
@@ -74,6 +74,7 @@
     </dependency>
   </dependencies>
 
+
```

Comment:
This looks like a leftover. Shall we clean up this?

Reply:
Please ignore this - I'll revert this commit after the testing is done